.github/scripts/summarize_tests.sh

```diff
@@ -5,6 +5,7 @@ set -eEuo pipefail
 unit_test_file="./build-and-test-output/unit_tests_output.txt"
 integration_test_file="./build-and-test-output/integration_tests_output.txt"
 coverage_report_file="./coverage-output/coverage_report.json"
+e2e_test_file="./e2e-test-output/e2e_test_output.txt"
```

Comment:
It's out of the scope of this PR, but maybe it would be better to have a single outputs directory and then create inside it all the reports. 
I think a good pattern would be:
```
unit_test_file="./outputs/unit_tests_report.txt"
integration_test_file="./outputs/integration_tests_report.txt"
coverage_report_file="./outputs/coverage_report.json"
e2e_test_file="./outputs/e2e_test_report.txt"
```

Reply:
Ok makes sense, addressed inside last commit.

Reply:
No unfortunately I am having some issues trying to quickly fix this; I'll addrss it in a later PR.
Thanks